cloudinit/cmd/query.py

```diff
@@ -65,6 +78,26 @@ def get_parser(parser=None):
     return parser
 
 
+def load_userdata(ud_file_path):
+    """Attempt to return a string of user-data from ud_file_path
+
+    Attempt to decode or decompress if needed.
+    If unable to decode the content, raw bytes will be returned,
+    which load_json base64-encodes, adding a ci-b64: prefix to the value when
+    emitting the JSON output.
```

Comment:
This is confusing to me; I see no load_json here nor any prefix appending to the content.

Reply:
I think I should drop this content from the docstring. This base64 encoding context is really something we do before we return a serializing json blob in cloud-init query output, it doesn't apply to this specific function. I've added the same-ish docstring content to the module __doc__ where it makes a bit more sense.